public/locales/en-GB/grunnlag.json

```diff
@@ -1,13 +1,13 @@
 {
     "INNTEKT_GRUNNLAG_DESCRIPTION": "* The earnings for this year are based on your pensionable income from {{year}}.",
     "UFORE_GRUNNLAG_DESCRIPTION": "* The earnings for this year are based on the fact that you had {{uforegrad}} disability benefits in {{year}}.",
-    "FORSTEGANGSTJENESTE_GRUNNLAG_DESCRIPTION": "* Earnings for this year are based on your first-time military service in {{year}}.",
+    "FORSTEGANGSTJENESTE_GRUNNLAG_DESCRIPTION": "* Earnings for this year are based on your compulsory military service in {{year}}.",
```

Comment:
Hvorfor bytter "first/time" med "compulsory"

Reply:
To grunner:

- Det er "standard" oversettelse, ref. https://www.dinordbok.no/norsk-engelsk/?q=f%C3%B8rstegangstjeneste
- Andre steder i Opptjening-appen brukes "compulsory military service"